cyberark-secrets-provider-for-kubernetes/templates/_helpers.tpl

```diff
@@ -0,0 +1,18 @@
+{{/*
+Return the most recent RBAC API available
+*/}}
+{{- define "conjur.rbac-api" -}}
+{{- if .Capabilities.APIVersions.Has "rbac.authorization.k8s.io/v1" }}
+{{- printf "rbac.authorization.k8s.io/v1" -}}
+{{- else if .Capabilities.APIVersions.Has "v1" }}
+{{- printf "v1" -}}
```

Comment:
Since you use the `{{- blah -}}` pattern everywhere you can use regular indentation to make the logic blocks not looks so cluttered

Reply:
I wanted to follow the[ helm/chart](https://github.com/helm/charts/blob/886cac277eaaada4ac2c1495c81a11f025c588b9/incubator/solr/templates/_helpers.tpl) pattern as they have it